packages/beeq/src/components/date-picker/_storybook/bq-date-picker.stories.tsx

```diff
@@ -0,0 +1,288 @@
+  };
+
+  return html`
+    <div class="w-[280px]">
+      <bq-date-picker
+        ?autofocus=${args.autofocus}
+        clear-button-label=${args['clear-button-label']}
+        distance=${args.distance}
+        ?disable-clear=${args['disable-clear']}
+        ?disabled=${args.disabled}
+        form=${ifDefined(args.form)}
+        min=${ifDefined(args.min)}
+        max=${ifDefined(args.max)}
+        locale=${ifDefined(args.locale)}
+        ?show-outside-days=${args['show-outside-days']}
+        first-day-of-week=${args['first-day-of-week']}
+        name=${ifDefined(args.name)}
+        ?open=${args.open}
+        panel-height=${args['panel-height']}
+        placeholder=${args.placeholder}
+        placement=${args.placement}
+        ?required=${args.required}
+        skidding=${args.skidding}
+        strategy=${args.strategy}
+        validation-status=${args['validation-status']}
+        type=${args.type}
+        months=${args.months}
+        value=${ifDefined(args.value)}
+        .format-options=${args['format-options']}
+        .isDateDisallowed=${isDate}
+        @bqBlur=${args.bqBlur}
+        @bqChange=${args.bqChange}
+        @bqClear=${args.bqClear}
+        @bqFocus=${args.bqFocus}
+      >
+        ${!args.noLabel ? label : nothing}
+        ${args.prefix ? html`<bq-icon name="user-circle" slot="prefix"></bq-icon>` : nothing}
+        ${args.suffix ? html`<bq-icon name="arrow-down" slot="suffix"></bq-icon>` : nothing}
+      </bq-date-picker>
+    </div>
+  `;
+};
+
+export const InitialValue: Story = {
+  render: (args) => html`
+    <div class="flex flex-row gap-4">
+      <!-- Default date picker -->
+      <div class="flex flex-col gap-2">
+        <p>Default date picker</p>
+        ${Template({ ...args, value: '2024-05-25', name: 'bq-date-picker-default', noLabel: 'true' })}
+      </div>
+      <!-- Range date picker -->
+      <div class="flex flex-col gap-2">
+        <p>Range date picker</p>
+        ${Template({
+          ...args,
+          value: '2024-06-04/2024-06-15',
+          name: 'bq-date-picker-range',
+          type: 'range',
+          months: 2,
+          noLabel: 'true',
+        })}
+      </div>
+      <!-- Multi date picker -->
+      <div class="flex flex-col gap-2">
+        <p>Multi date picker</p>
+        ${Template({
+          ...args,
+          value: '2024-05-08 2024-05-09 2024-05-10',
+          name: 'bq-date-picker-multi',
+          type: 'multi',
+          months: 2,
+          noLabel: 'true',
+        })}
+      </div>
+    </div>
+  `,
+};
+
+export const MixMax: Story = {
+  name: 'Min and Max allowed dates',
+  render: Template,
+  args: {
+    min: '2024-06-05',
+    max: '2024-06-15',
+  },
+};
+
+export const Disabled: Story = {
+  render: Template,
+  args: {
+    disabled: true,
+  },
+};
+
+export const ValidationStatus: Story = {
+  name: 'Validation',
+  render: (args) => html`
+    <div class="flex flex-row gap-4">
```

Comment:
We can use a grid instead

```suggestion
    <div class="grid grid-cols-1 gap-m sm:grid-cols-3">
```

Reply:
@Cata1989 I think this 👆🏼  is not solved yet